src/test/java/uk/gov/hmcts/sscs/service/reminder/HearingHoldingReminderRemoverTest.java

```diff
@@ -55,9 +61,10 @@ public void canHandleEvent() {
     }
 
     @Test
-    public void removedHearingHoldingReminderWhenHearingBooked() {
```

Comment:
Would the test name not be better as it was? Think it is good to try and follow a given/when/then pattern where possible in unit tests